src/libs/feature-previews-config.ts

```diff
@@ -81,12 +82,21 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
     feedbackUrl: `mailto:dsp-sue@broadinstitute.org?subject=${encodeURIComponent('Feedback on Hail Batch (Azure)')}`,
   },
   {
-    id: ENABLE_AZURE_COLLABORATIVE_WORKFLOWS,
-    title: 'Azure Collaborative Workflows',
+    id: ENABLE_AZURE_COLLABORATIVE_WORKFLOW_READERS,
+    title: 'Azure Collaborative Workflows Read-only',
     description:
-      'Enabling this feature will allow for workspaces to become collaborative with other users to run workflows and read workspace data',
+      'Enabling this feature will allow for Azure workspaces to become collaborative with other users to read workflow data run by workspace owner',
     feedbackUrl: `mailto:dsp-workflow-management@broadinstitute.org?subject=${encodeURIComponent(
-      'Feedback on Azure Collaborative Workflows experience.'
+      'Feedback on Azure Collaborative Workflows experience, Phase 1.'
+    )}`,
+  },
+  {
+    id: ENABLE_AZURE_COLLABORATIVE_WORKFLOW_RUNNERS,
```

Comment:
Is this unused currently?

Reply:
Yes, this PR is just putting it in place for future tickets.